include/abb_librws/rws_client.h

```diff
@@ -386,10 +398,16 @@ class RWSClient : public POCOClient
    * \brief A method for retrieving the current robtarget values of a mechanical unit.
    *
    * \param mechunit for the mechanical unit's name.
+   * \param tool for the tool frame relative to which the robtarget will be reported.
+   * \param wobj for the wobj relative to which the robtarget will be reported.
+   * \param coordinate for the coordinate mode (base, world, tool, or wobj) in which the robtarget will be reported.
    *
    * \return RWSResult containing the result.
    */
-  RWSResult getMechanicalUnitRobTarget(const std::string& mechunit);
+  RWSResult getMechanicalUnitRobTarget(const std::string& mechunit,
+                                       const Coordinate& coordinate = RWSClient::Coordinate::ACTIVE,
```

Comment:
```suggestion
                                       const Coordinate& coordinate = ACTIVE,
```
Related to scoped enum.